app/javascript/src/components/ConsultationList.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchReservations } from '../redux/reservation/reservationsThunk';
+import { reservationsState } from '../redux/reservation/reservationsSlice';
+
+function ConsultationList() {
```

Comment:
- [ ] OPTIONAL: Whenever I add a new consultation, if I visit the "My consultation" section, it won't appear:

![image](https://github.com/MohammadYaser/appointment-app/assets/57421823/2498c671-c927-46ac-8ad1-790084599994)

After checking the rails and Chrome consoles, I can asses that it isn't indeed loading anything.

The only way to make the consultations appear is by reloading the page:

![image](https://github.com/MohammadYaser/appointment-app/assets/57421823/3ff8957d-df28-468b-92b5-1ad9dcf90cb6)

I highly suggest to use a useEffect statement so whenever a user visits "My consultations" it will indeed check the server for the updated consultations lists, so they display without having to reload the page. 👍